vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -113,6 +116,16 @@
     @Nullable
     private VimeoAccount mVimeoAccount;
 
+    public interface Caller<DataType_T> {
```

Comment:
This is following a style pattern that we agreed on in one of our previous Android syncs:
https://github.vimeows.com/MobileApps/Style-Guides/blob/master/Android/README.md

Open for debate of course. Feel free to weigh in on it or add it to a future sync agenda if you strongly disagree.